src/Common/TMS.Common/Errors/AppError.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Net;
+
+namespace TMS.Common.Errors;
+
+public class AppError
```

Comment:
AppError - seems to be naming is not clear. That should be something like an API error or HTTP error because that exception type wrap only http failure codes.

Reply:
renamed to ApiError